src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -741,4 +762,40 @@
     <value>Open your settings.json file. Alt+Click to open your defaults.json file.</value>
     <comment>{Locked="settings.json"}, {Locked="defaults.json"}</comment>
   </data>
+  <data name="ColorScheme_DeleteButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Delete the current color scheme.</value>
+  </data>
+  <data name="ColorScheme_Rename.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Rename</value>
+  </data>
+  <data name="Profile_DeleteButtonDisclaimerInBox" xml:space="preserve">
+    <value>This profile cannot be deleted because it is included by default.</value>
+  </data>
+  <data name="Profile_DeleteButtonDisclaimerDynamic" xml:space="preserve">
+    <value>This profile cannot be deleted because it is automatically generated.</value>
+  </data>
+  <data name="ColorScheme_DeleteButtonDisclaimerInBox" xml:space="preserve">
+    <value>This color scheme cannot be deleted or renamed because it is included by default.</value>
+  </data>
+  <data name="ColorScheme_DeleteConfirmationButton.Content" xml:space="preserve">
+    <value>Yes, delete color scheme</value>
+  </data>
+  <data name="ColorScheme_DeleteConfirmationMessage.Text" xml:space="preserve">
+    <value>Are you sure you want to delete this color scheme?</value>
+  </data>
+  <data name="ColorScheme_RenameAccept.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Accept rename</value>

```

Comment:
We're going to have multiple versions of these, so I'd call it `Accept` and `Cancel` and make them use a common `Uid` (not one that's namespaced in ColorScheme)`